staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -22,6 +22,19 @@ limitations under the License.
 // leader (a.k.a. fencing). A client observes timestamps captured locally to
 // infer the state of the leader election. Thus the implementation is tolerant
 // to arbitrary clock skew, but is not tolerant to arbitrary clock skew rate.
+// Timestamp(renew time) is not meaningful if it was collected on another
+// machine. The implementation of this client only acts on locally collected
+// timestamps and cannot rely on the accuracy of timestamp in the record for
+// correctness.
+//
+// A client needs to wait a full LeaseDuration without observing a change
```

Comment:
This is probably better placed in the LeaseDuration field documentation

Reply:
Will do.